src/navigation/CollapsingNavigationMenu.module.scss

```diff
@@ -24,13 +26,13 @@
     padding-top: 0.7143rem;
 
     &.is-selected {
-      background-color: #e1f3fd;
+      background-color: var(--terra-dev-site-collapsing-navigation-menu-item-background-color, #e1f3fd);
       box-shadow: inset 0 0 0 0.07143rem #fff;
     }
 
     @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none), (hover: hover) and (pointer: fine) {
       &:hover {
-        background-color: #e1f2fd;
+        background-color: var(--terra-dev-site-collapsing-navigation-menu-item-hover-background-color, #e1f3fd);
```

Comment:
This might be a record on variable length

Reply:
Should I try to shorten it a little? These scss variable tend to be quite verbose due to a need to avoid collisions. Also, if this truly is a record I'm expecting a trophy.

Reply:
I wouldn't shorten it. With how theming works the variable length isn't particularly important. As long as it's unique and descriptive it should be fine.

Reply:
![](https://media.giphy.com/media/26u4exk4zsAqPcq08/giphy.gif)